docs/source/guide/webhooks.md

```diff
@@ -0,0 +1,143 @@
+```json
+{
+    "action": "TASK_CREATED",
+    "tasks": [
+        {"id": 1, ...},
+        ...
+    ]
+}
+```
+
+Your application or service can then respond to that event information however you want. 
+
+## What to use Label Studio webhooks for 
+
+Some events that happen in Label Studio happen in response to an API call. For those, the API response is sufficient to know that the action has occurred. But other events happen inside the Label Studio UI, such as project changes and task annotation activities. To find out immediately when those changes occur, you can use webhooks to notify your application or machine learning pipeline to take specific actions. 
+
+The actions your webhook takes in your application or pipeline can vary depending on the event. For example, you might use events sent to your webhook URL to do the following:
+- Start training a machine learning model after a certain number of tasks have been annotated.
+- Perform active learning after a task has been annotated.
+- Prompt annotators to begin working on a project after it is fully set up.
+- Create a new version of training data in a dataset versioning repository. 
+
+Limit the number of requests to your server by subscribing only to the events relevant for your use case.
+
+## Available Label Studio webhooks
+Label Studio makes two main types of events available to integrate with webhooks: project-level task events and organization events.
+
+<table>
+  <tr>
+    <th>Event Action</th>
+    <th>Details</th>
+    <th>Use Case</th>
+  </tr>
+  <tr>
+    <td>[Task Created](webhook_reference.html#Task-Created)</td>
+    <td>For a specific project, triggers when new tasks are created. One event per task.</td>
+    <td>Use to take action in your machine learning pipeline. </td>
+  </tr>
+  <tr>
+    <td>[Task Deleted](webhook_reference.html#Task-Deleted)</td>
+    <td>For a specific project, triggers when tasks are deleted. One event per deleted task.</td>
+    <td>Use to update a training dataset version. </td>
+  </tr>
+  <tr>
+    <td>[Annotation Created](webhook_reference.html#Annotation-Created)</td>
+    <td>For a specific project, triggers when new annotations are created for any tasks. One event per annotation.</td>
+    <td>Use to start training in an active learning scenario.</td>
+  </tr>
+  <tr>
+    <td>[Annotation Updated](webhook_reference.html#Annotation-Updated)</td>
+    <td>For a specific project, triggers when an existing annotation is updated, overwritten, or when a task is skipped.</td>
+    <td>Use to prompt model retraining. </td>
+  </tr>
+  <tr>
+    <td>[Annotation Deleted](webhook_reference.html#Annotation-Deleted)</td>
+    <td>For a specific project, triggers when an annotation is deleted.</td>
+    <td>Use to create a new version of a training dataset. </td>
+  </tr>
+  <tr>
+    <td>[Project Created](webhook_reference.html#Project-Created)</td>
```

Comment:
how do the project/organization-level events get exposed?

Reply:
1. Create a section on how to enable org webhooks by settings `LABEL_STUDIO_ALLOW_ORGANIZATION_WEBHOOKS=true`
2. Refer to this section in `Project  Created` and `Project Deleted`

Reply:
I am on it